src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReadThreadingAssemblerArgumentCollection.java

```diff
@@ -23,7 +24,11 @@
      */
     @Advanced
     @Argument(fullName="kmerSize", shortName="kmerSize", doc="Kmer size to use in the read threading assembler", optional = true)
-    public List<Integer> kmerSizes = Arrays.asList(10, 25);
+    public List<Integer> kmerSizes = new ArrayList<>();
```

Comment:
I think we didn't have an Arrays.asList() serializer registered which was causing kryo failures.  I added that but forgot to revert this change.